DataFormats/BTauReco/interface/TemplatedSecondaryVertexTagInfo.h

```diff
@@ -30,15 +30,15 @@ namespace reco {
     }
 
     struct TrackData {
-      enum Status { trackSelected = 0, trackUsedForVertexFit, trackAssociatedToVertex };
-
-      inline bool usedForVertexFit() const { return (int)svStatus >= (int)trackUsedForVertexFit; }
-      inline bool associatedToVertex() const { return (int)svStatus >= (int)trackAssociatedToVertex; }
+      static const int trackSelected = 0;
+      static const int trackUsedForVertexFit = 1;
+      static const int trackAssociatedToVertex = 2;
```

Comment:
@emilbols , please use `constexpr` instead of `const` ( http://cms-sw.github.io/cms_coding_rules.html#7--design-and-coding-guidelines-1 )